components/breadcrumb/stories/template.js

```diff
@@ -48,9 +47,13 @@ export const Template = ({
 								idx !== arr.length - 1,
 								() =>
 									html`<div
-										class="${rootClass}-itemLink"
+										class=${classMap({
+											[`${rootClass}-itemLink`]: true,
+											"is-disabled": isDisabled,
+										})}
+										?disabled=${isDisabled}
 										role="link"
-										tabindex="0"
+										tabindex="${isDisabled ? "" : 0}"
```

Comment:
```suggestion
										tabindex=${ifDefined(!isDisabled ? "0" : undefined)}
```
So the attribute does not render for non-disabled.

Reply:
Ah yes, this is what I was trying to do! (get it to not render) I forgot about `ifDefined`. Thanks!!